src/index.ts

```diff
@@ -278,7 +278,7 @@ export async function install(
           },
         },
         rollupPluginNodeResolve({
-          mainFields: ['browser', 'module', !isStrict && 'main'].filter(Boolean),
+          mainFields: ['module', 'jsnext:main', 'browser', !isStrict && 'main'].filter(Boolean),
```

Comment:
`jsnext:main` is not really a standard thing, and seems to be replaced entirely by `module`. But maybe this would help in supporting older libraries migrate?